compute/compute-node.Dockerfile

```diff
@@ -1916,26 +1916,31 @@ RUN apt update && \
       ;; \
     esac && \
     apt install --no-install-recommends -y \
+        bpftrace \
```

Comment:
> Some useful debugging tools are missing from the compute image and sometimes it's impossible to install them because memory is tightly packed.

Let's do that, it will slightly increase the image size, but I recall several people hitting this problem, e.g. @myrrc recently

I wonder what's the image size diff between one from this PR, e.g. `ghcr.io/neondatabase/vm-compute-node-v16:14043229281` vs. `main`? I don't think it matters much, but can you check and add to RP description just to be mindful of a size overhead

Reply:
Good point. I had a look at the qcow2 files inside and it's main=1.4g vs PR=1.6g. Right now I'm trying to mount them to see which new package came with so baggage.

Reply:
Right, it's bpftrace that needs over 160mb because it needs an LLVM toolchain and some extra libs.

Reply:
Ok, I dropped bpftrace for now. We can add it later. I put the image difference into the PR description.